sklearn/metrics/tests/test_ranking.py

```diff
@@ -38,7 +38,6 @@
 
 def make_prediction(dataset=None, binary=False):
     """Make some classification predictions on a toy dataset using a SVC
-
```

Comment:
Please don't remove these blank lines. 

More generally: 
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.